lib/postprocess/image.js

```diff
@@ -93,6 +95,19 @@ module.exports = async files => {
           response.headers['content-type'] || response.headers['Content-Type']
         imageExtensions = contentType.split('/')[1]
         imageBuffer = Buffer.from(response.data)
+
+        // check if the file is valid
+        try {
+          sizeOf(imageBuffer)
+        } catch (e) {
+          if (e instanceof RangeError) {
+            const imageSrc = fs.readFileSync(path.join(__dirname, 'missing-image.png')).toString('base64')

```

Comment:
I would recommend preloading the placeholder in any case. This way we are fetching it from disk each time..